packages/docs/src/pages/components/upload/examples/ExDragDrop.vue

```diff
@@ -32,17 +32,19 @@
     </section>
 </template>
 
-<script>
-    export default {
+<script lang="ts">
+    import { defineComponent } from 'vue'
+
+    export default defineComponent({
```

Comment:
Please explicitly import and register the Buefy components used in this component. For instance, https://github.com/ntohq/buefy-next/blob/14b4f83d11b50711df427333e01fb8195f36827a/packages/docs/src/pages/components/autocomplete/examples/ExSimple.vue#L20-L27

Otherwise, no type checking is performed for Buefy components.